backend/typescript/services/implementations/authService.ts

```diff
@@ -177,26 +180,28 @@ class AuthService implements IAuthService {
     }
 
     try {
-      const passwordResetLink = await firebaseAdmin
-        .auth()
-        .generatePasswordResetLink(email);
-      // first-time determines if we are setting a new account password
-      // (account will be verified) or reseting an old account's password
-      const setPasswordLink = passwordResetLink.concat("&first-time=true");
+      const authId = await this.userService.getAuthIdById(user.id);
+
+      const setPasswordLink = `${process.env.CLIENT_URL}/auth/action?mode=verify-user&uid=${authId}`;
 
       const emailBody = `
       Hello,
       <br><br>
       You have been invited to join CCBC as a ${user.roleType.toLowerCase()}. Please use the link 
```

Comment:
The copy here is a bit misleading since it says to use the a link but you actually have a hyperlink which is linked by text. Users might get confused trying to search for an actual url

Reply:
I think we'll have to coordinate this change with the designers since I got this from the designers last term. I'll reach out to Sadia and the designers to see if they have a different copy in mind

Reply:
Updated the copy based on the feedback from designers